API/Migrations/OtrContextModelSnapshot.cs

```diff
@@ -18,6 +18,9 @@ protected override void BuildModel(ModelBuilder modelBuilder)
 #pragma warning disable 612, 618
             modelBuilder
                 .HasAnnotation("ProductVersion", "8.0.1")
+                .HasAnnotation("Proxies:ChangeTracking", false)
+                .HasAnnotation("Proxies:CheckEquality", false)
+                .HasAnnotation("Proxies:LazyLoading", true)
```

Comment:
Is this intentional?

Reply:
Not sure how this got generated..

Reply:
The LazyLoading may have been generated by having `virtual` on Match but that isn't new.

Reply:
Oh I think it's from this `.UseLazyLoadingProxies()`